common/client/versionChecker.go

```diff
@@ -40,11 +40,11 @@ const (
 	CLI = "cli"
 
 	// SupportedGoSDKVersion indicates the highest go sdk version server will accept requests from
-	SupportedGoSDKVersion = "1.6.0"
+	SupportedGoSDKVersion = "1.7.0"
 	// SupportedJavaSDKVersion indicates the highest java sdk version server will accept requests from
-	SupportedJavaSDKVersion = "1.6.0"
+	SupportedJavaSDKVersion = "1.7.0"
```

Comment:
I see the Java feature version update to 1.4.0. Do we use the same version for Go and Java?

Reply:
Interesting. I have no idea why it was set to 1.6 before. Then Looks like I should set it to `1.5.0` for the new release?

Reply:
Set to `1.4.0`